scripts/webpack/prod.servers.config.js

```diff
@@ -13,9 +14,17 @@ const SERVER_ROOT = path.join(PROJECT_ROOT, 'packages', 'server')
 const GQL_ROOT = path.join(PROJECT_ROOT, 'packages', 'gql-executor')
 const SFU_ROOT = path.join(PROJECT_ROOT, 'packages', 'sfu')
 const DOTENV = path.join(PROJECT_ROOT, 'scripts/webpack/utils/dotenv.js')
-const publicPath = getWebpackPublicPath()
 const distPath = path.join(PROJECT_ROOT, 'dist')
 
+const getNormalizedWebpackPublicPath = () => {
+  let publicPath = getWebpackPublicPath()
```

Comment:
Question: why do we even return a protocol relative url from `getWebpackPublicPath`? the only other place where this function is used is in `getS3BasePath`, which also normalizes the url (but to http).

Reply:
tbh i can't even remember! maybe we don't need it? IIRC Jordan was the one who wrote that code. it's that old! things have probably changed